wolox-books/src/app/screens/sign-up/sign-up.component.scss

```diff
@@ -0,0 +1,75 @@
+@import './src/scss/colors';
+
+.content {
```

Comment:
you can set this to a flex column to reduce the properties used in the child elements